web/src/components/share-buttons/share-buttons.tsx

```diff
@@ -69,7 +69,7 @@ function ShareButtons({ getString, shareTextId }: Props) {
         target="_blank"
         rel="noopener noreferrer"
         onClick={() => trackSharing('twitter', locale)}>
-        <FontIcon type="twitter" />
+        <XICon className=""/>
```

Comment:
You can remove the `className` prop. It's an empty string and it does not look like the component accepts this prop